### PR TITLE
Clean up config rollup config

### PR DESCRIPTION
**Pull request by stramel**
Keep a single copy of the config and utilize the package name for the files.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -1,29 +1,42 @@
...
+  output = outputConfig,
+} = {}) {
+  return {
+    input: 'src/index.ts',
+    external: ['react', 'react-dom'],
+    plugins: [
+      typescript({
+        tsconfig,
+      }),
+      terser({
```
Comment (bluebill1049) [inline rollup.config.js]: We only want this for ie11 build right?

Comment (stramel) [reply rollup.config.js]: Nah, we should do it for all. It won't hurt anything to minify the code. Most libraries do.

Comment (bluebill1049) [reply rollup.config.js]: interesting :) let's see if it exceed 5kb hehe

Comment (bluebill1049) [reply rollup.config.js]: think you are right hehe

**Pull request merged**